apps/fhir/bluebutton/utils.py

```diff
@@ -712,7 +712,7 @@ def handle_http_error(e):
     return e
 
 
-def build_fhir_response(request, call_url, cx, r=None, e=None):
+def build_fhir_response(request, call_url, crosswalk, r=None, e=None):
```

Comment:
want to switch out `r` and `e` while you're in here?

Reply:
I'm going to leave this one for now, just because these PRs always make me more nervous than they need to, especially as they get bigger. I'm going to try to do a number of these PRs over the next couple weeks, so this will go on my list